precompiles/avs/IAVSManager.sol

```diff
@@ -13,6 +13,40 @@ IAVSManager constant AVSMANAGER_CONTRACT = IAVSManager(
 /// @dev The interface through which solidity contracts will interact with AVS-Manager
 /// @custom:address 0x0000000000000000000000000000000000000901
 interface IAVSManager {
+    event AVSRegistered(address indexed sender, string avsName, bool success);
+    event AVSUpdated(address indexed sender, string avsName, bool success);
+    event AVSDeregistered(address indexed sender, string avsName, bool success);
+    event OperatorJoined(address indexed sender, bool success);
+    event OperatorOuted(address indexed sender, bool success);
+    event TaskCreated(address indexed sender,uint64 taskId,address indexed taskContractAddress,string name,bytes hash,
+        uint64 taskResponsePeriod,uint64 taskChallengePeriod,uint64 thresholdPercentage,uint64 taskStatisticalPeriod
+    );
+    event ChallengeInitiated(address indexed sender, bytes taskHash, uint64 taskID, bytes taskResponseHash,
+        string operatorAddress, bool success);
+    event PublicKeyRegistered(address indexed sender, string name, bool success);
+    event OperatorRegisteredToExocore(address indexed sender, string metaInfo, bool success);
+    event TaskSubmittedByOperator(address indexed sender, uint64 taskID, string taskResponse,
+        string blsSignature, string taskContractAddress, string stage, bool success);
+
+
+    struct TaskInfo {
+        string taskContractAddress;
+        string name;
+        bytes hash;
+        uint64 taskId;
+        uint64 taskResponsePeriod;
+        uint64 taskStatisticalPeriod;
+        uint64 taskChallengePeriod;
+        uint64 thresholdPercentage;
+        uint64 startingEpoch;
+        uint64 actualThreshold;
+        string[] optInOperators;
+        string[] signedOperators;
+        string[] noSignedOperators;
+        string[] errSignedOperators;
+        string taskTotalPower;
```

Comment:
This has too many members and will probably error in Solidity.

Reply:
fixed